packages/node-core/src/configure/migration-service/migration.ts

```diff
@@ -2,76 +2,177 @@
 // SPDX-License-Identifier: GPL-3.0
 
 import {SUPPORT_DB} from '@subql/common';
-import {getAllEntitiesRelations, GraphQLEntityField, GraphQLEntityIndex, GraphQLModelsType} from '@subql/utils';
-import {IndexesOptions, ModelAttributes, ModelStatic, Sequelize, Transaction, Utils} from '@subql/x-sequelize';
+import {
+  getAllEntitiesRelations,
+  GraphQLEntityField,
+  GraphQLEntityIndex,
+  GraphQLEnumsType,
+  GraphQLModelsType,
+  GraphQLRelationsType,
+  hashName,
+  IndexType,
+} from '@subql/utils';
+import {
+  IndexesOptions,
+  ModelAttributes,
+  ModelStatic,
+  QueryTypes,
+  Sequelize,
+  Transaction,
+  Utils,
+} from '@subql/x-sequelize';
 import {GraphQLSchema} from 'graphql';
-import Pino from 'pino';
+import {isEqual} from 'lodash';
 import {StoreService} from '../../indexer';
-import {formatAttributes, formatColumnName, generateHashedIndexName, modelToTableName, syncEnums} from '../../utils';
-import {getColumnOption, modelsTypeToModelAttributes} from '../../utils/graphql';
+import {getLogger} from '../../logger';
 import {
```

Comment:
The naming of these functions is inconsistent, some end in `Query` others in `Statement` and some have nothing.